config/migrations/2024/bestuurseenheden/20240919101332-add-new-welzijnsvereniging-sint-anna/20240919101332-add-new-organisation-sint-anna.sparql

```diff
@@ -0,0 +1,61 @@
+        <http://www.w3.org/ns/regorg#orgStatus>	<http://lblod.data.gift/concepts/63cc561de9188d64ba5840a42ae8f0d6> ;
+        <http://data.vlaanderen.be/ns/besluit#classificatie>	<http://data.vlaanderen.be/id/concept/BestuurseenheidClassificatieCode/e8294b73-87c9-4fa2-9441-1937350763c9> .
+
+      # Bestuursorganen voor Sint-Anna
+      <http://data.lblod.info/id/bestuursorganen/40d628b0-6c5d-11ef-8f29-2728b827bd59> <http://data.vlaanderen.be/ns/besluit#bestuurt>	<http://data.lblod.info/id/bestuurseenheden/66DB132B6ABCA6E6BF2DBBED> .
+      <http://data.lblod.info/id/bestuursorganen/40bfba80-6c5d-11ef-8f29-2728b827bd59> <http://data.vlaanderen.be/ns/besluit#bestuurt>	<http://data.lblod.info/id/bestuurseenheden/66DB132B6ABCA6E6BF2DBBED> .
+      <http://data.lblod.info/id/bestuursorganen/40cab700-6c5d-11ef-8f29-2728b827bd59> <http://data.vlaanderen.be/ns/besluit#bestuurt>	<http://data.lblod.info/id/bestuurseenheden/66DB132B6ABCA6E6BF2DBBED> .
+
+      # Insert bestuursorganen
+      <http://data.lblod.info/id/bestuursorganen/40d628b0-6c5d-11ef-8f29-2728b827bd59> rdf:type	<http://data.vlaanderen.be/ns/besluit#Bestuursorgaan> ;
+        <http://mu.semte.ch/vocabularies/core/uuid>	"40d628b0-6c5d-11ef-8f29-2728b827bd59" ;
+        skos:prefLabel "Leidend Ambtenaar Welzijnsvereniging Sint-Anna" ;
+        <http://data.vlaanderen.be/ns/besluit#classificatie>	<http://data.vlaanderen.be/id/concept/BestuursorgaanClassificatieCode/41caf7e6-b040-4720-9cc2-a96cfffed5b4> ;
+        <http://data.vlaanderen.be/ns/besluit#bestuurt>	<http://data.lblod.info/id/bestuurseenheden/66DB132B6ABCA6E6BF2DBBED> .
+
+      <http://data.lblod.info/id/bestuursorganen/40bfba80-6c5d-11ef-8f29-2728b827bd59> rdf:type	<http://data.vlaanderen.be/ns/besluit#Bestuursorgaan> ;
+	      <http://mu.semte.ch/vocabularies/core/uuid>	"40bfba80-6c5d-11ef-8f29-2728b827bd59" ;
+        skos:prefLabel "Raad van bestuur Welzijnsvereniging Sint-Anna" ;
+	      <http://data.vlaanderen.be/ns/besluit#classificatie>	<http://data.vlaanderen.be/id/concept/BestuursorgaanClassificatieCode/013cc838-173a-4657-b1ae-b00c048df943> ;
+	      <http://data.vlaanderen.be/ns/besluit#bestuurt>	<http://data.lblod.info/id/bestuurseenheden/66DB132B6ABCA6E6BF2DBBED> .
+
+      <http://data.lblod.info/id/bestuursorganen/40cab700-6c5d-11ef-8f29-2728b827bd59> rdf:type	<http://data.vlaanderen.be/ns/besluit#Bestuursorgaan> ;
+        <http://mu.semte.ch/vocabularies/core/uuid>	"40cab700-6c5d-11ef-8f29-2728b827bd59" ;
+        skos:prefLabel "Algemene vergadering Welzijnsvereniging Sint-Anna" ;
+        <http://data.vlaanderen.be/ns/besluit#classificatie>	<http://data.vlaanderen.be/id/concept/BestuursorgaanClassificatieCode/b52094ff-21a2-4da8-8dbe-f513365d1528> ;
+        <http://data.vlaanderen.be/ns/besluit#bestuurt>	<http://data.lblod.info/id/bestuurseenheden/66DB132B6ABCA6E6BF2DBBED> .
+
+      # Insert bestuursorganen in tijd
+      <http://data.lblod.info/id/bestuursorganen/40ec21b0-6c5d-11ef-8f29-2728b827bd59>
```

Comment:
The bestuursfunctie needs to be added:
```sparql
<http://data.lblod.info/vocabularies/leidinggevenden/heeftBestuursfunctie> <http://data.lblod.info/id/bestuursfuncties/40eee0d0-6c5d-11ef-8f29-2728b827bd59>
```

We then need to add the bestuursfunctie URI separately as well as it doesn't seem to be present in Loket yet:
```sparql
<http://data.lblod.info/id/bestuursfuncties/40eee0d0-6c5d-11ef-8f29-2728b827bd59>
  <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://data.lblod.info/vocabularies/leidinggevenden/Bestuursfunctie> ;
  <http://mu.semte.ch/vocabularies/core/uuid> "40eee0d0-6c5d-11ef-8f29-2728b827bd59" ;
  <http://www.w3.org/ns/org#role> <http://data.vlaanderen.be/id/concept/BestuursfunctieCode/855489b9-b584-4f34-90b2-39aea808cd9f> .
```

`<http://data.vlaanderen.be/id/concept/BestuursfunctieCode/855489b9-b584-4f34-90b2-39aea808cd9f>` is present in Loket, and its properties and objects don't need to be added separately.